src/Resources/values-tw/strings.xml

```diff
@@ -5,7 +5,7 @@
 	<string name="app_titlebar_title_receive">從您的 Windows 10 / 11 電腦接收資料</string>
 	<string name="app_titlebar_title_receive_setup">設定與資訊</string>
 
-	<string name="assets_prefix">zh</string>
+	<string name="assets_prefix">zh-tw</string>
```

Comment:
Based on your description you might want to use `zh-Hant-TW`
> Type: redundant
Tag: zh-Hant-TW
Description: Taiwan Chinese in traditional script
Added: 2005-04-11